tests/test_logger.py

```diff
@@ -0,0 +1,139 @@
+"""Test the context-enabled log adapter."""
+
+import logging
+import unittest
+
+from muselog import context
+from muselog.logger import get_logger_with_context
+
+from .support import ClearContext
+
+LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+class LoggerTestCase(ClearContext, unittest.TestCase):
```

Comment:
Thanks for writing these tests it makes it really clear on what functionality is being added